index.js

```diff
@@ -0,0 +1,45 @@
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+import BookCollection from './modules/bookCollection.js';
+import addNavigation from './modules/addNavigation.js';
+
+const datePlaceholder = document.getElementById('current-date');
+
+datePlaceholder.innerHTML = DateTime.local().toLocaleString(DateTime.DATETIME_FULL);
+
+const bookListWrapper = document.getElementById('bookList');
+const addBookForm = document.getElementById('addBookForm');
+const bookCollection = new BookCollection(bookListWrapper);
+
+addBookForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const id = Date.now().toString();
```

Comment:
- [ ] Date is not displayed correctly. Kindly ensure that the current **date is displayed** below the navigation bar, using the [Luxon](https://github.com/moment/luxon/blob/master/docs/install.md) library.

![image](https://user-images.githubusercontent.com/12745474/153853429-ce960b38-be01-47b3-91b0-e4228d804f0b.png)

Reply:
@rloterh This is also because of the same issue I mentioned above. No bulid script and therefore error in JS and of course this placeholder doesn't render the correct date and time.

Reply:
@ahangarha 
I actually cloned and run your project locally, and the results were the same as your GitHub version. The issue is not with npm. Kindly ensure to go through your code thoroughly.

Reply:
Did you run `npm install` to install dependencies?

I want to know how to install Luxon and import it to the project in a standard way and still the live demo works as we expect. No way!  Me and four other students have spent 3 hours to find a way to follow the specifications and still get the expected result. We failed in finding one.

I would really appreciate if you provide some hint.

Reply:
@ahangarha 
Following our discussion on slack, there are some discrepancies with the project description which we anticipate it's fixed soon, so as to prevent future students from experiencing the same issue. Thanks for pointing out.

Attention: @Davidosky007
https://questions.microverse.org/t/how-to-import-and-use-luxon-without-webpack/2319/6

Reply:
Thank you @rloterh for following up.